graylog2-server/src/test/java/org/graylog/testing/containermatrix/annotations/ContainerMatrixTestsConfiguration.java

```diff
@@ -60,13 +60,13 @@
      * matrix rule
      * If no version is explicitly specified, then {@link SearchServer#DEFAULT_VERSION will be used by the tests}
```

Comment:
Maybe remove the comment and the DEFAULT_VERSION generally when we by default use all of them?